### DISCUSSION TITLE
Kibana is NOT Populating Logs from SO 2.4.5???

### DISCUSSION THREAD

**Discussion by cyberdaddy93** [Security-Onion-Solutions/securityonion#11050] [2.4]
### Few things I've noticed after the installation of SO 2.4
+Kibana does not automatically populate logs from SO.
+Alerts are NOT being generated on the SOC Dashboard. 

**NOTE**
Enabled Zeek and Suricata by going to SOC Dashboard Administration > Configuration > Kibana/firewall/zeek/etc...

**Comment by dougburks** [comment]
From https://docs.securityonion.net/en/2.4/architecture.html#heavy-node:
> Heavy nodes are NOT recommended for most users

Is there any particular reason you need a heavy node? Have you tried using a normal forward node instead?

For more information about a normal distributed deployment including a forward node, please see:
https://docs.securityonion.net/en/2.4/architecture.html#distributed

**Reply by cyberdaddy93** [reply]
I realize the heavy nodes are not **recommended** for most users, however it does not say you can't use it. Heavy Nodes works best for our deployment and has been great before 2.4.5 release.

**Reply by defensivedepth** [reply]
There is currently a known issue with Heavy Nodes, where Suricata alerts are not being correctly ingested.

This issue is tracking that fix: https://github.com/Security-Onion-Solutions/securityonion/issues/11031

**Reply by cyberdaddy93** [reply]
It's not just the heavynodes, I tried with a forward node and a sensor node. Kibana is not showing zeek logs or any network stats. I feel this integration with Elastic Agents is a fumble. Not a fan of 2.4.5 so far.

**Comment by dougburks** [comment]
> I feel this integration with Elastic Agents is a fumble. Not a fan of 2.4.5 so far.

First, please review our discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and specificially this:
**_Please be patient, courteous, and respectful. Disrespectful messages can result in being banned._**

Also keep in mind that 2.4.5 was pre-release software. We've since released 2.4.10 and a hotfix yesterday:
https://blog.securityonion.net/2023/08/security-onion-24-has-reached-general.html
https://blog.securityonion.net/2023/08/security-onion-2410-hotfix-20230821-now.html

> It's not just the heavynodes, I tried with a forward node and a sensor node. Kibana is not showing zeek logs or any network stats.

If you're having problems with standard forward nodes as well, then have you double-checked that you're actually receiving traffic from your tap or span port? Is Zeek actually writing logs to `/nsm/zeek/logs/current/`?

At this point, I'd recommend downloading the latest ISO image that we released yesterday and performing a fresh installation:
https://blog.securityonion.net/2023/08/security-onion-2410-hotfix-20230821-now.html

If you have further questions or problems, please start a new discussion.

**Reply by cyberdaddy93** [reply]
"_Also keep in mind that 2.4.5 was pre-release software. We've since released 2.4.10 and a hotfix yesterday:_"

So does this mean this issue is fixed with this release?

"If you're having problems with standard forward nodes as well, then have you double-checked that you're actually receiving traffic from your tap or span port? Is Zeek actually writing logs to /nsm/zeek/logs/current/?"

Of course I did, and yes zeek is collecting logs just fine. Hence, why I said the logs are not being forwarded to Kibana.

"First, please review our discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and specificially this:
Please be patient, courteous, and respectful. Disrespectful messages can result in being banned."

If my comment came across "Disrespectful" 

I apologize, sincerely. I really do.  I'll try the new 2.4.10 and hopefully it doesn't get me mad enough to bring me back on here to get me banned. 
- . -.  ..-. ..- -.-. -.-  -.-- --- ..-  -.-. .-. -.--  -... .- -... -.--
